python/paddle/v2/fluid/layer_helper.py

```diff
@@ -21,19 +21,11 @@ def name(self):
 
     @property
     def main_program(self):
-        prog = self.kwargs.get('main_program', None)
-        if prog is None:
-            return default_main_program()
-        else:
-            return prog
+        return default_main_program()
```

Comment:
As we can change the default program, I think it's improper to keep calling it `default`.

Reply:
maybe just `main_program()` is cool. It could be done in the following PR